build/platform.props

```diff
@@ -0,0 +1,45 @@
+<Project>
+  <PropertyGroup>
+    <!-- directories -->
+    <_WorkRoot>$(RepositoryRoot).w\</_WorkRoot>
+    <_WorkLayoutDir>$(_WorkRoot).l\</_WorkLayoutDir>
+    <_WorkOutputDir>$(_WorkRoot).o\</_WorkOutputDir>
+    <_SrcDir>$(RepositoryRoot)src\</_SrcDir>
+    <_AllMetapackageDir>$(_SrcDir)Microsoft.AspNetCore.All\</_AllMetapackageDir>
+    <_TemplatesDir>$(MSBuildThisFileDirectory)tools\templates\</_TemplatesDir>
+    <_DockerDir>$(MSBuildThisFileDirectory)tools\docker\</_DockerDir>
+    <_PackagingDir>$(MSBuildThisFileDirectory)tools\packaging\</_PackagingDir>
+    <_DebToolDir>$(MSBuildThisFileDirectory)tools\dotnet-deb-tool-consumer\</_DebToolDir>
+    <_SFSourceDir>$(RepositoryRoot).deps\Signed\SF\</_SFSourceDir>
+    <_InstallerSourceDir>$(RepositoryRoot).deps\Installers\</_InstallerSourceDir>
+    <_DockerRootDir>/opt/code/</_DockerRootDir>
+    <_InstallersOutputDir>$(ArtifactsDir)installers\</_InstallersOutputDir>
+
+    <!-- installers -->
+    <SFInstallerName>aspnetcore-shared</SFInstallerName>
+    <HostingInstallerName>dotnet-hosting</HostingInstallerName>
+    <DebConfigFile>$(_PackagingDir)debian_config.json</DebConfigFile>
+
+    <PublicCoreFeedPrefix>https://dotnetcli.blob.core.windows.net/dotnet</PublicCoreFeedPrefix>
```

Comment:
Need to update this to consume `DotNetAssetRootUrl` when building in orchestrated-build.